db/seeds/app.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const { getTopics } = require("../controllers/getTopics.controller");
+console.log('in the app');
+const app = express();
+
+app.use(express.json()); //?
```

Comment:
this is only needed when making POST requests. remove it for now, then add it back in when needed.